src/Miso/Concurrent.hs

```diff
@@ -11,6 +11,9 @@
 module Miso.Concurrent (
     Notify (..)
   , newNotify
+  , newEmptyNotify
+  , EventWriter (..)
```

Comment:
Where is this coming from? This doesn’t look like it will actually compile.

Reply:
Needed to rebase